test/surface_test.py

```diff
@@ -2394,27 +2395,28 @@ def test_premul_surf(src_col,
                                            src_size=(17, 67),
                                            dst_size=(69, 69)
                                            ))
-
-        self.assertEqual(*test_premul_surf(pygame.Color(30, 20, 0, 255),
-                                           pygame.Color(40, 20, 0, 51),
-                                           src_size=(17, 67),
-                                           dst_size=(69, 69),
-                                           src_has_alpha=False,
-                                           ))
-        self.assertEqual(*test_premul_surf(pygame.Color(30, 20, 0, 51),
-                                           pygame.Color(40, 20, 0, 255),
-                                           src_size=(17, 67),
-                                           dst_size=(69, 69),
-                                           dst_has_alpha=False,
-                                           ))
-        self.assertEqual(*test_premul_surf(pygame.Color(30, 20, 0, 255),
-                                           pygame.Color(40, 20, 0, 255),
-                                           src_size=(17, 67),
-                                           dst_size=(69, 69),
-                                           src_has_alpha=False,
-                                           dst_has_alpha=False,
-                                           ))
-
+        # These tests trigger some of the weird SDL1 alpha blending behaviour
```

Comment:
It makes me a little nervous that these aren't correct on SDL1. Are the tests correct, or is SDL1 incorrect? Has the behaviour changed in these blit routines now? Do the SDL2 ones not match the SDL1 ones?

Reply:
This doesn't need a response necessarily :) I was just thinking out loud.

Reply:
No, you are right to ask I'm definitely going to poke at it and see if I can figure out why this works differently.

I mean in theory you don't _need_ to do pre-multiplied alpha blending in any of those cases because both surfaces don't have alpha but I believe I would expect it to just work boringly without going through the fancy intrinsic code....

Anyway, it just didn't seem the moment to go off on a tangent figuring that out in this PR.